ecs-cli/modules/cli/regcreds/regcreds_app.go

```diff
@@ -14,11 +14,32 @@
 package regcreds
 
 import (
+	"fmt"
+
+	secretsClient "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/secretsmanager"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
 	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils/regcreds"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/secretsmanager"
+	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
 	"github.com/urfave/cli"
 )
 
+const (
+	// EcsCliResourcePrefix is prepended to the names of resources created through the ecs-cli
+	// TODO: move to more generic package for use in other parts of the ecs-cli
+	EcsCliResourcePrefix = "amazon-ecs-cli-setup-"
+)
+
+// CredsOutputEntry contains the credential ARN and associated container names
+// TODO: use & move to output_reader once implemented?
+type CredsOutputEntry struct {
```

Comment:
Wait wasn't this added in #598? Is this PR intentionally on top of those commits?

Reply:
The code that's in the 3 new commits LGTM.

Reply:
yeah, once #598 gets merged in this will be a much smaller set of changes.